catalog/solr/catalog-solr-core/src/test/java/ddf/catalog/source/solr/provider/SolrProviderQuery.java

```diff
@@ -766,37 +763,6 @@ public void testContextualAnyText() throws Exception {
         1, getFilterBuilder().attribute(Metacard.ANY_TEXT).is().like().text(soughtWord), provider);
   }
 
-  @Test
-  public void testQueryExcludedAttributes() throws Exception {
```

Comment:
❓  Not sure how meaningful the implications are of not sending the spell check option to Solr. Worth writing a test for it?

Reply:
Not sending it is the default.  It is what we were doing before the spell check PR.